intbot/core/integrations/pretalx.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any
+
+import httpx
+from core.models import PretalxData
+from django.conf import settings
+
+PRETALX_EVENT = "ep2025"
+base_url = f"https://pretalx.com/api/events/{PRETALX_EVENT}/"
```

Comment:
It would be nice if this would be in some config/env file, so we can dynamically fetch different data without having to change anything in the code. I know that previous versions are deployed and it's static data, but for example once we'll want to switch to 2026, then we'd have to update this file as well, instead of just one config.